client/me/account-password/index.jsx

```diff
@@ -171,3 +172,8 @@ module.exports = React.createClass( {
 		);
 	}
 } );
+
+export default connect(
+	null,
+	dispatch => bindActionCreators( { errorNotice }, dispatch )
```

Comment:
Hmm, I didnt think about it much... I wanted to keep it consistent with other files, where we have more actions in props.